package-lock.json

```diff
@@ -1,9 +1,7297 @@
 {
   "name": "ufo-explorer",
   "version": "0.1.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
what's your node version? This change should be ok.. Although we should pin node version so it does not switch between dev env